### PR TITLE
Add support for Microsoft PowerPoint template ( `.potx`)

### LINKED ISSUE #709: Consider adding `opendocument` and `officedocument` templates mime ?
### Description

Hello, at this time, it seems templates from ms office and openoffice/libreoffice templates are considered either `application/zip` or at best as there document counter parts. So I was wondering if it would be possible to better analyze these files. For example, using [mime](https://www.npmjs.com/package/mime), it detects these mime types for the following extensions:

- [ ] `ots: application/vnd.oasis.opendocument.spreadsheet-template` (application/zip for file-type) #718
- [ ] `odg: application/vnd.oasis.opendocument.graphics` (application/zip for file-type) #718
- [ ] `otg: application/vnd.oasis.opendocument.graphics-template` (application/zip) #718
- [ ] `otp: application/vnd.oasis.opendocument.presentation-template`(application/zip for file-type) #718
- [ ] `ott: application/vnd.oasis.opendocument.text-template` (application/zip for file-type) #718

- [x] `xltx: application/vnd.openxmlformats-officedocument.spreadsheetml.template` (application/vnd.openxmlformats-officedocument.spreadsheetml.sheet for file-type) #712 
- [x] `xltm: application/vnd.ms-excel.template.macroenabled.12` (application/vnd.openxmlformats-officedocument.spreadsheetml.sheet for file-type) #714
- [x] `potx: application/vnd.openxmlformats-officedocument.presentationml.template` (application/vnd.openxmlformats-officedocument.presentationml.presentation for file-type) #710 
- [x] `dotx: application/vnd.openxmlformats-officedocument.wordprocessingml.template` (application/vnd.openxmlformats-officedocument.wordprocessingml.document for file-type) #713 

I don't know much about mime types or file signatures, so I don't know they are good candidates, neither how to do the detections.

### Existing Issue Check

- [X] I have searched the existing issues and could not find any related to my problem.

### File-Type Scope Acknowledgment

- [X] I understand that file-type detects binary file types and not text or other formats.

### PR DESCRIPTION
**Pull request by sly7-7**
Add support for  the **Microsoft PowerPoint template** file format:
- Extension: `potx`
- MIME-type:  `application/vnd.openxmlformats-officedocument.presentationml.template` [^1]

Partially resolves #709

[^1]: https://learn.microsoft.com/en-us/previous-versions/office/office-2007-resource-kit/ee309278%28v=office.12%29

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -529,6 +529,7 @@ abortController.abort(); // Abort file-type reading from the Blob stream.
 - [`pdf`](https://en.wikipedia.org/wiki/Portable_Document_Format) - Portable Document Format
 - [`pgp`](https://en.wikipedia.org/wiki/Pretty_Good_Privacy) - Pretty Good Privacy
 - [`png`](https://en.wikipedia.org/wiki/Portable_Network_Graphics) - Portable Network Graphics
+- [`potx`](https://en.wikipedia.org/wiki/List_of_Microsoft_Office_filename_extensions) - Microsoft Powerpoint template
```
Comment (Borewit) [inline readme.md:532]:
```suggestion
- [`potx`](https://en.wikipedia.org/wiki/List_of_Microsoft_Office_filename_extensions) - Microsoft PowerPoint template
```

**Review by Borewit — CHANGES_REQUESTED** [review]

**Comment by Borewit** [comment]
Thanks for your contribution @sly7-7, your code is looking good. 
Please use the initial comment for the PR description.

**Comment by sly7-7** [comment]
Thanks for your feedback, I will do the changes tomorrow, I'm away from keyword right now.

**Review by Borewit — APPROVED** [review]
LGTM

**Pull request merged**